contracts/core/EntryPoint.sol

```diff
@@ -25,8 +26,8 @@ contract EntryPoint is IEntryPoint, StakeManager {
 
     SenderCreator private immutable senderCreator = new SenderCreator();
 
-    // internal value used during simulation: need to query aggregator if wallet is created
-    address private constant SIMULATE_NO_AGGREGATOR = address(1);
+    // internal value used during simulation: need to query aggregator.
+    address private constant SIMULATE_FIND_AGGREGATOR = address(1);
```

Comment:
Unrelated but I suggest using a completely random address like `0xfefefefe...` because `address(1)` or something like that is actually used for precompiles which may be confusing.